languages/liveblog.pot

```diff
@@ -1,60 +1,60 @@
-# Copyright (C) 2016 Liveblog
+# Copyright (C) 2018 Liveblog
 # This file is distributed under the same license as the Liveblog package.
 msgid ""
 msgstr ""
-"Project-Id-Version: Liveblog 1.5\n"
+"Project-Id-Version: Liveblog 1.8.2\n"
```

Comment:
This will need to be `1.9.0` - generating should be done during the release branch.